diff --git a/sites/all/modules/rules/includes/faces.inc b/sites/all/modules/rules/includes/faces.inc
index a1fc93bd..812d96a1 100644
--- a/sites/all/modules/rules/includes/faces.inc
+++ b/sites/all/modules/rules/includes/faces.inc
@@ -224,6 +224,7 @@ if (!class_exists('FacesExtendable', FALSE)) {
      *   describe a single file.
      */
     public function extend($interface, array $callbacks = array(), array $includes = array()) {
+      $interface = class_exists($interface) ? $interface : FALSE;
       $face_methods = $interface ? get_class_methods($interface) : array_keys($callbacks);
       if ($interface) {
         if (array_diff($face_methods, array_keys($callbacks))) {
